config/routes.rb

```diff
@@ -3,4 +3,9 @@
   resources :users, only: [:index, :show] do
     resources :posts, only: [:index, :show]
   end
+  get "/posts/new", to: "posts#new"
+  post "posts/new", to: "posts#create"
+  get '/comments/new', to: "comments#new"
+  post '/comments/new', to: "comments#create"
+  post '/likes/new', to: "likes#create"
```

Comment:
- [ ] Please modify your routes to take into account that a like belongs to a given post.
You should have something like `post '/posts/:post_id/likes`.